pysat/tests/test_instrument.py

```diff
@@ -12,9 +13,9 @@
 
 import pysat
 import pysat.instruments.pysat_testing
-import pysat.instruments.pysat_testing_xarray
 import pysat.instruments.pysat_testing2d
 import pysat.instruments.pysat_testing2d_xarray
+import pysat.instruments.pysat_testing_xarray
```

Comment:
according to https://www.niso.org/sites/default/files/2017-08/tr03.pdf symbols like underscore come before numbers in alphabetical order. I think the original location was correct.

Reply:
I would agree, but the linter says it's wrong. I'm leaning toward leaving this rule on, as it's a minor issue for me, but open to input from the dev team.

Reply:
I don't care